activesupport/lib/active_support/number_helper/number_to_currency_converter.rb

```diff
@@ -7,21 +7,25 @@ module NumberHelper
     class NumberToCurrencyConverter < NumberConverter # :nodoc:
       self.namespace = :currency
 
-      def convert
-        number = self.number.to_s.strip
+      def convert(number = self.number)
+        number = number.to_s.strip
         format = options[:format]
 
         if number.to_f.negative?
           format = options[:negative_format]
           number = absolute_value(number)
         end
 
-        rounded_number = NumberToRoundedConverter.convert(number, options)
+        rounded_number = number_to_rounded_converter.execute(number)
         format.gsub("%n".freeze, rounded_number).gsub("%u".freeze, options[:unit])
       end
 
       private
 
+        def number_to_rounded_converter
```

Comment:
This method is only called once in this class for each instance, how the memoizing at instance level improve the performance here?

Reply:
If you're reusing the parent instance, then this memoized instance gets reused as well.